domains/Settings/ManageReligion/Services/UpdateReligionPosition.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Settings\ManageReligion\Services;
+
+use App\Interfaces\ServiceInterface;
+use App\Models\Religion;
+use App\Services\BaseService;
+use Illuminate\Support\Facades\DB;
+
+class UpdateReligionPosition extends BaseService implements ServiceInterface
```

Comment:
This kind of service (that handle position of items in a list) already exist a bunch of times. Don't you want to consider using a more generic solution instead of duplicate code each time?

Reply:
That'd be great, but how can we make it generic?